### PR TITLE
[ppi]: Enable bulk API

### PR DESCRIPTION
**Pull request by nazariig**
Signed-off-by: Nazarii Hnydyn <nazariig@nvidia.com>

**HLD:** https://github.com/sonic-net/SONiC/pull/1084

* fix vslib to handle bulk requests properly
* enable bulk API for both VS and NVIDIA

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@kcudnik , @saiarcot895 could you please help to review? 
@nazariig  please check coverage report and have it increased.

**Comment by nazariig** [comment]
FYI. `vstest` is unstable: each run is a totally different story.

**Review by saiarcot895 — DISMISSED** [review]

**Comment by nazariig** [comment]
@kcudnik can you please have a look?

**Review by saiarcot895 — APPROVED** [review]

File: vslib/VirtualSwitchSaiInterface.cpp [hunk 1]
```diff
@@ -1298,6 +1298,12 @@ sai_status_t VirtualSwitchSaiInterface::bulkCreate(
 {
     SWSS_LOG_ENTER();
 
+    // create new real object IDs
+    for (uint32_t idx = 0; idx < object_count; idx++)
+    {
+        object_id[idx] = m_realObjectIdManager->allocateNewObjectId(object_type, switch_id);
+    }
+
```
Comment (kcudnik) [inline vslib/VirtualSwitchSaiInterface.cpp:1306]: was that missing or this is already done in bulkCreate( generic in line 1310 ?

Comment (nazariig) [reply vslib/VirtualSwitchSaiInterface.cpp:1306]: @kcudnik it was missing, this part has been taken from single create method

**Pull request merged**